animations/index.md

```diff
@@ -1,159 +1,88 @@
-classes.
-
-### Staggered animation
+__________________________________  
 
-Animations that are broken into smaller motions, where some of the motion is delayed.
-The smaller animations may be sequential, or may partially or completely overlap.
-
-* [Staggered Animations](/animations/staggered-animations/) <img src="/images/ic_new_releases_black_24px.svg" alt="this doc is new!"> NEW<br>
+Well-designed animations makes a UI more intuitive, contributes to the sophisticated look and feel of a polished app, and improves the user experience. Flutter’s animation framework makes it easy to implement basic and advanced animation types—so whether you are new to animations or an animation wizard, Flutter provides the platform to bring your creative vision to life.
 
-## Other resources
-
-Learn more about Flutter animations at the following links:
+Using the built-in Flutter animation widgets means you don’t have to write sophisticated drawing code. Flutter animation widgets include definitions for standard motion effects such *fade out* or *slide transition* so most of the code required to draw each animation frame is already written for you. And Flutter includes a built-in graphics rendering and animation infrastructure for both iOS and Android so you build one animation for both platforms.
 
-* [Animations: Technical Overview](/animations/overview.html)<br>
-A look at some of the major classes in the animations library,
-and Flutter's animation architecture.
+With only a few lines of code, you can create attractive and effective animations using the implicit and transition widgets. And if you want to add complex animations, Flutter also provides  explicit animation widgets for customized effects so you can build your own animations using the `AnimatedBuilder` or `TransitionBuilder` widgets.
 
-* [Animation and Motion Widgets](/widgets/animation/)<br>
-A catalog of some of the animation widgets provided in the Flutter APIs.
+## Flutter animation widgets
+All Flutter animations include these basic properties:
+1. Animations must be associated with an object to animate.
+2. Animations must define what type of animation will be performed.
+3. Animations must define how long the animation will last.
 
-{% comment %}
-Until the landing page for the animation library is reworked, leave this
-link out.
-* The [animation
-library](https://docs.flutter.io/flutter/animation/animation-library.html)
-in the [Flutter API documentation](https://docs.flutter.io/)<br>
-The animation API for the Flutter framework.
-{% endcomment %}
+The Flutter SDK animation widgets form the basis for rendering your animation on iOS and Android platforms. Flutter animations widgets are *implicit*, *transition*, or *explicit* animations. If you’re starting out in animations, or you want to quickly add an animation to your app, try out the implicit or transition animation widgets.
 
-<hr>
+![Flutter animations](images/code_complexity3.png)
```

Comment:
What does “Build” mean as the label for the Y axis? I would label the Y axis as “flexibility and expressiveness” and the X axis as "implementation complexity." As mentioned above, it would be great to have an accompanying table to lay out how these two dimensions change as the developer moves from one type of animation implementation to another.